test/fixtures/rules/std_rule_cases/L061.yml

```diff
@@ -0,0 +1,17 @@
+rule: L061
+
+test_pass_not_equal_to:
+  pass_str: |
+    SELECT * FROM X WHERE 1 != 2
```

Comment:
As discussed above, also include a unit test like:

```yml
test_pass_not_equal_to_tsql:
  pass_str: |
    SELECT * FROM X WHERE 1 !   = 2
  configs:
    core:
      dialect: tsql
```

and 

```yml
test_fail_not_equal_to_tsql:
  fail_str: |
    SELECT * FROM X WHERE 1  <
      -- some comment
    > 2
  fix_str: |
    SELECT * FROM X WHERE 1  !
      -- some comment
    = 2
  configs:
    core:
      dialect: tsql
```
(n.b there are already other rule that focus on removing intermediate whitespace so we're just concerned with doing a straight swap here)